txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) (isAddressValid bool) {
```

Comment:
Why is the return value named if it isn't used?

Reply:
Using a named return is used to eliminate the requirement of mentioning the variable name with the return statement.

Reply:
What variable name? You're not returning an variable, just values

Reply:
updated